src/bio/bio_xstream.c

```diff
@@ -466,7 +458,7 @@ xs_poll_completion(struct bio_xs_context *ctxt, unsigned int *inflights,
 
 struct spdk_blob_store *
 load_blobstore(struct bio_xs_context *ctxt, char *bdev_name, uuid_t *bs_uuid,
-	       bool create, bool async,
+	       bool create, bool async, unsigned int roles,
```

Comment:
Why don't you pass 'cluster_size' directly? Anyway, I don't think we should use different cluster size since the data blob and meta blob could reside in same device/blobstore.